Code/Source/RobotImporter/URDF/RobotImporter.h

```diff
@@ -19,12 +22,19 @@ namespace ROS2
         AZStd::string prefabFilePath;
     };
 
-    namespace RobotImporter
+    class RobotImporter
     {
-        void Import(
-            const RobotImporterConfig& config,
-            std::function<void(const AZStd::string&)> infoLogger,
-            std::function<void(const AZStd::string&)> errorLogger);
-    }
+    public:
+        RobotImporter(std::function<void(const AZStd::string&)> infoLogger,
+                      std::function<void(const AZStd::string&)> errorLogger);
+        void Import(const RobotImporterConfig& config);
+        void Update(std::function<void()> importFinishedCb);
```

Comment:
As above - this is unclear

Reply:
Changed the name to be more informative and added docs